wiki/Beatmaps/id.md

```diff
@@ -0,0 +1,77 @@
+# Beatmaps 
+
+*Tidak bingung dengan: [Kesulitan](/wiki/Difficulties).*
+
+Beatmaps adalah tingkat kesulitan dalam [osu!](/wiki/osu!_glossary), sama seperti tahapan di [Permainan DS](/wiki/Glossary/#ds-games). Dalam bentuk paling dasar mereka, beatmap memiliki lagu dan berbagai [hit objects](/wiki/hit_objects) yang ditempatkan sesuai dengan irama musik.Mereka juga dapat memiliki gambar latar belakang, video, bahkan [storyboard](/wiki/storyboard).
+
+Beatmaps diunggah menggunakan [BSS](/wiki/BSS) (Beatmap Submission System) dan kemudian melalui proses persetujuan yang disebut [modding](/wiki/modding) untuk mempertahankan standar.
+
+Anda dapat menemukan beatmaps di [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) di halaman di osu! web.
```

Comment:
remove `di` after `halaman`, it's ambiguous.
also remove `s` after `Beatmaps` in `Beatmaps Listing`, or translate it.